internal/uri/uri_util.go

```diff
@@ -19,16 +19,19 @@ package uri
 import (
 	"fmt"
 	"math"
+	"net/url"
 	"os"
 	"path/filepath"
 	"regexp"
+	"runtime"
 	"strings"
 
 	sglsp "github.com/sourcegraph/go-lsp"
 	"go.lsp.dev/uri"
 )
 
 const fileScheme = "file://"
+const UncFileScheme = "file:////"
```

Comment:
Does this need to be exported?

```suggestion
const uncFileScheme = "file:////"
```